### PR TITLE
[Term Entry] Python SQL Connectors: SQLite3

### LINKED ISSUE #5635: [Term Entry] Python SQL Connectors: SQLite3
### Reason/inspiration (optional)

We would like a new entry on the `sqlite3` term under Python. The entry should go in a new file under `docs/content/python/concepts/sql-connectors/terms/sqlite3/sqlite3.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ralfze**
### Description

- Added term sqlite3 for Python SQL Connector
- Closes #5635

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @ralfze ,
The entry is nicely written. I had a look at your file and suggested a few changes please make them asap and let me know.

Thanks,
Savi

**Comment by ralfze** [comment]
Hey @SaviDahegaonkar,
thank you very much for the feedback.
I added the suggested edits and pushed it in a new commit.

Best,
Ralf

File: content/python/concepts/sql-connectors/terms/sqlite3/sqlite3.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+The **`.commit()`** function persists the inserted values to the database.
+
+```py
+con.commit()
+```
+
+## Check the inserted rows
+
+In order to check all inserted rows, the function **`.fetchall()`** can be used.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/sql-connectors/terms/sqlite3/sqlite3.md:90]:
```suggestion
To check all inserted rows, the  **`.fetchall()`**  function can be used.
```

File: content/python/concepts/sql-connectors/terms/sqlite3/sqlite3.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+```py
+result = cursor.execute("SELECT * FROM persons")
+
+result.fetchall()
+```
+
+## Close the connection
+
+After all transactions are completed, the connection should be closed with **`.close()`**.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/sql-connectors/terms/sqlite3/sqlite3.md:100]:
```suggestion
After completing all the transactions , connection can be closed with **`.close()`**.
```

File: content/python/concepts/sql-connectors/terms/sqlite3/sqlite3.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+```py
+connection.close()
+```
+
+## Codebyte Example
+
+The complete example for tinkering.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/sql-connectors/terms/sqlite3/sqlite3.md:108]:
```suggestion
"An example showing how to connect to an `SQLite` database, `create a table`, `insert/query data`, and `close the connection.`"
```

**Review by SaviDahegaonkar — APPROVED** [review]

**Comment by ralfze** [comment]
I updated the code that caused the prettier code test to fail.

**Review by Sriparno08 — APPROVED** [review]
Looks good, @ralfze!

**Comment by ralfze** [comment]
> Looks good, @ralfze!

Thank you @Sriparno08!

**Pull request merged**